### ISSUE TITLE
Add `tee` dependency to Dockerfile for container images

### ISSUE THREAD

**Issue by sourishkrout** [smallstep/certificates#360]
Otherwise some subcommand will break

**Comment by tashian** [comment]
I'm curious how this bug came about? `tee` is part of Alpine's mini root filesystem, it should be available to any `golang:alpine` container

**Comment by maraino** [comment]
@sourishkrout I don't think this is a good idea, we can add `tee`, but the main problem is that you also need `sudo` access, and we don't want that.

**Comment by maraino** [comment]
The command that is failing is not intended to be run on that image, you're supposed to run it on your laptop/desktop. A better solution if we want to run that getting started in docker, would be to use a different image, or create everything using docker-compose.
